app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -43,21 +44,34 @@
 
     SharedPreferences sharedpreferences;
     SharedPreferences.Editor editor;
+    private Button okButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_trustees);
 
-        System.out.println(comradeEditText.size());
-        comradeEditText.add((EditText) findViewById(R.id.comrade1EditText));
```

Comment:
I think a temporary variable is created either way. Creating a temporary variable and adding it to the list also happens on two clock cycles.
However, as you mentioned, I feel this is not required currently. I shall undo the changes